fog/api/proto/view.proto

```diff
@@ -145,6 +145,11 @@ message QueryResponse {
     /// This can be used by the client as a hint when choosing cryptonote mixin indices.
     /// This field doesn't have the same "cursor" semantics as the other fields.
     uint64 last_known_block_cumulative_txo_count = 9;
+
+    /// If true, this means that due limits, we could not return all the requested
+    /// user events in one response. Clients cannot compute an accurate balance check
```

Comment:
In the PR description you write: "I believe that this is a backwards compatible change, because old clients will ignore this flag".

But if it's true that "clients cannot compute an accurate balance check until they have received all relevant user events," then how can an (old) client that ignores this flag accurately compute a balance check?

Reply:
Good question:

* Today, the amount of user events in postgres is not so much that we can't return all the events, and so all the existing clients are fine, and will be fine with servers before and after this change.
* After this change, we can ship clients that will be okay even when that is no longer the case, and the servers will have to do some sort of pagination.
* The backwards compatible part is, new clients that talk to old servers, will get all the events, and the flag will default to false when they deserialize, so they will correctly decide that they got all the events.